api/src/Controller/BaseController.php

```diff
@@ -80,6 +80,10 @@
  *          type="integer",
  *      )
  *  )
+ *
+ *  @OA\Schema(
```

Comment:
Also just curious, what was the message you saw before adding this? What was the testing package looking for here?

Reply:
I get a lot of `Warning: $ref "#/components/schemas/BaseController" not found for @OA\Schema() in api/src/Controller/BaseController.php on line 5` errors and the yaml still includes BaseController in all the schemas, which then fail to load because BaseController is not defined anywhere.